src/views/Arys/ActionButtons/AryCopyModal/index.tsx

```diff
@@ -0,0 +1,249 @@
+import React, { useState, useCallback, useMemo } from 'react';
+import { connect } from 'react-redux';
+import { isDefined } from '@togglecorp/fujs';
+
+import Modal from '#rscv/Modal';
+import ModalHeader from '#rscv/Modal/Header';
+import ModalBody from '#rscv/Modal/Body';
+import ModalFooter from '#rscv/Modal/Footer';
+import LoadingAnimation from '#rscv/LoadingAnimation';
+import ChecklistInput from '#rsci/ChecklistInput';
+import SearchInput from '#rsci/SearchInput';
+
+import Button from '#rsu/../v2/Action/Button';
+
+import {
+    currentUserAdminProjectsSelector,
+} from '#redux';
+import {
+    RequestClient,
+    methods,
+} from '#request';
+
+import notify from '#notify';
+import _ts from '#ts';
+import _cs from '#cs';
+
+import {
+    notifyOnFailure,
+    notifyOnFatal,
+} from '#utils/requestNotify';
+
+import {
+    AddRequestProps,
+    Requests,
+    AppState,
+} from '#typings';
+
+import styles from './styles.scss';
+
+interface OwnProps {
+    className?: string;
+    closeModal?: () => void;
+    userProjects: ProjectItem[];
+    assessmentId: number;
+    projectId: number;
+}
+
+interface ProjectItem {
+    id: number;
+    title: string;
+    assessmentTemplate?: number;
+}
+
+interface PropsFromAppState {
+    userProjects: ProjectItem[];
+}
+
+interface Body {
+    assessments: number[];
+    projects: number[];
+}
+
+interface AssessmentsByProjects {
+    [key: string]: number[];
```

Comment:
The correct typings would be 
```suggestion
    [key: string]: number[] | undefined;
```